frontend/src/app/components/containers/complaints/outcomes/hwcr-complaint-assessment.tsx

```diff
@@ -229,12 +229,15 @@ export const HWCRComplaintAssessment: FC = () => {
           key: selectedJustification?.label,
           value: selectedJustification?.value,
         },
-        assessment_type: selectedAssessmentTypes?.map((item) => {
-          return {
-            key: item.label,
-            value: item.value,
-          };
-        }),
+        assessment_type:
+          selectedActionRequired?.label === "No"
```

Comment:
I think that if we're using "Yes" and "No" values these should be treated as constants so that they can be reused else where, makes things less error prone